package.json

```diff
@@ -15,7 +15,8 @@
   },
   "heroku-run-build-script": true,
   "dependencies": {
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "vue-awesome": "^3.5.4"
```

Comment:
discussed in hub-discuss, gonna remove this in favor of the css that we already have from @fortawesome/fontawesome-free